homeassistant/components/venstar/__init__.py

```diff
@@ -41,6 +41,7 @@ async def async_setup_entry(hass: HomeAssistant, config: ConfigEntry) -> bool:
 
     venstar_data_coordinator = VenstarDataUpdateCoordinator(
         hass,
+        config,
```

Comment:
May be we should rename `config` to `config_entry` here

Reply:
you're right :+1: